### ISSUE TITLE
Remove trailing slash from default lang URL's

### ISSUE THREAD

**Issue by ouun** [polylang/polylang#224]
## Prerequisites

- [x] This is not a usage question (Those should be directed to the [community supported forum](https://wordpress.org/support/plugin/polylang), unless this is a question about Polylang Pro in which case you should [use the helpdesk](https://polylang.pro/support/)).
- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest `master` branch of Polylang and the latest WordPress version.
- [x] This bug happens with only Polylang plugin active
- [x] This bug happens with a default WordPress theme active
- [x] I can reproduce this bug consistently

## Steps to reproduce the issue

1. Wordpress redirects the main URL of an website from example.com/ (with trailing slash) to example.com (without trailing slash)
2. Polylang returns the main URL with a trailing slash (example.com/) in language switcher as well as in: `<link rel="alternate" href="http://example.com/" hreflang="en">` for the default language.
3. This is not SEO-perfect as e.g. Google Search Console warns that the alternate language defined returns a 302 status code.

## Expected behavior and actual behavior

The URL of the default language is outputted without trailing slash as by WP default:
`<link rel="alternate" href="http://example.com" hreflang="en">`

Thank you and best regards!

**Comment by Chouby** [comment]
I initially thought it was a bug from Polylang, but it looks like WordPress itself adds this trailing slash. I tested on an install with WP 4.9.4 + Twenty Seventeen and no plugin activated. If you look at the home link in the header, it includes a trailing slash too.

**Comment by ouun** [comment]
Dear @Chouby,
Thank you very much for your reply. I just did the same: set-up a blank WP 4.9.4 environment without any plugins and standard theme. For me Safari & Firefox (Mac) load the website without a trailing slash (or they hide it). Chrome instead displays a trailing slash.
As I'm using Bedrock from Roots for dev I thought it might be related to that. But the behavior is still the same. So I need some more investigation to find the root of this behavior.

Thank you and best regards,

Philipp

**Comment by jiboune** [comment]
Dear @Chouby , @ooun 

I'm actually facing the same issue with a clean install and default theme (WordPress 5.5.3 using Twenty Twenty theme). I’ve been able to remove the trailing slash at without using Polylang (2.8.4) with htaccess rules : 

```
RewriteCond %{REQUEST_FILENAME} !-d
RewriteRule ^(.*)/$ /$1 [L,R] # <- for test, for prod use [L,R=301]
```

But when Polylang is activated the home url returned is /en/ or /fr/ depending on the selected language. The other pages works as expected without trailing slash after the slug. I also tried to set a redirection from /en/ to /en, I got an ERR_TOO_MANY_REDIRECTS error. 

Did you find a way to remove this trailing slash for the home page ?
 
Best regards
